wire-gradle-plugin/src/test/kotlin/com/squareup/wire/gradle/WirePluginTest.kt

```diff
@@ -186,17 +175,20 @@ class WirePluginTest {
     val result = gradleRunner.runFixture(fixtureRoot) { buildAndFail() }
 
     assertThat(result.task(":generateProtos")).isNull()
+    assertThat(result.output)
+      .containsPattern(
+        "Invalid path string: \".*src/main/proto/squareup/geology/period.proto\"",
+      )
     assertThat(result.output)
       .contains(
         """
-            |Invalid path string: "src/main/proto/squareup/geology/period.proto".
-            |For individual files, use the following syntax:
-            |wire {
-            |  sourcePath {
-            |    srcDir 'dirPath'
-            |    include 'relativePath'
-            |  }
-            |}
+        |For individual files, use the following syntax:
+        |wire {
+        |  sourcePath {
+        |    srcDir 'dirPath'
+        |    include 'relativePath'
```

Comment:
For those cases, I like using a syntax that works in both Groovy and Kotlin:

```suggestion
        |    srcDir("dirPath")
        |    include("relativePath")
```

It's less "idiomatic" in Groovy though so either work with me

Reply:
Great idea.

Reply:
https://github.com/square/wire/pull/2844